client/src/actions/availabilityActions.js

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+
+export const UPDATE_AVAILABILITY = 'UPDATE_AVAILABILITY'
+export const UPDATE_AVAILABILITY_FAIL = 'UPDATE_AVAILABILITY_FAIL'
+export const GET_AVAILABILITY = 'GET_AVAILABILITY'
+export const GET_AVAILABILITY_FAIL = 'GET_AVAILABILITY_FAIL'
+export const AVAILABILITY_LOADING = 'AVAILABILITY_LOADING'
+
+const baseURL = process.env.REACT_APP_SERVER_URL
+
+export const getAvailability = employeeId => dispatch => {
+  axios
+    .get(`${baseURL}/availabilities/${employeeId}`, {
+      headers: { authorization: 'testing' }
+    })
+    .then(res => {
+      dispatch({
+        type: GET_AVAILABILITY,
+        payload: res.data
+      })
+    })
+    .catch(error =>
+      dispatch({
+        type: GET_AVAILABILITY_FAIL,
+        payload: error
+      })
+    )
+}
+
+export const editAvailability = ({
+  changes,
+  availability
+}) => async dispatch => {
+  const { id: availabilityId, user_id: employeeId } = availability
+
+  try {
+    const res = await axios.put(
+      `${baseURL}/availabilities/${availabilityId}`,
+      changes,
+      {
+        headers: { authorization: 'testing' }
```

Comment:
I think we're moving off of using authorization: 'testing', so you'll want to get the token off of the connected form and send that through the action creator, and use that instead of 'testing'

Reply:
done